Sources/OTel/OTel+Bootstrap.swift

```diff
@@ -20,6 +20,94 @@ import Tracing
 // MARK: - API
 
 extension OTel {
+    /// Bootstrap observability backends with OTLP exporters.
+    ///
+    /// - Parameter configuration: Configuration for observability backends.
+    ///
+    ///   This value can be used to configure or disable the bootstrapped backends and defaults to
+    ///   `OTel.Configuration.default`, which bootstraps all backends with the default configuration defined in the
+    ///   OpenTelemetry specification.
+    ///
+    ///   Configuration can also be provided at runtime with environment variable overrides.
+    ///
+    /// - Returns: A service that manages the background work for the bootstrapped observability backends.
+    ///
+    ///   The returned service manages the complete lifecycle of all observability backends. It handles startup,
+    ///   background processing, and graceful shutdown of exporters and processors.  The service performs these
+    ///   operations.
+    ///
+    ///   > Important: You must run this service in a `ServiceGroup` alongside your application services for
+    ///   observability data to be exported.
+    ///
+    /// This is the primary API for setting up observability in your Swift application. Call this function once during
+    /// application startup to configure logging, metrics, and tracing with OTLP exporters.
+    ///
+    /// This function bootstraps the process-global observability subsystems that are enabled in the configuration.
+    /// Attempting to bootstrap these subsystems multiple times will result in a fatal error. If you wish to bootstrap
+    /// only a subset of the observability subsystems, provide a configuration with only those subsystems enabled.
+    ///
+    /// This API supports overriding the configuration using environment variables defined in the OpenTelemetry
+    /// specification. This enables operators to customize the observability of your application during deployment.
+    /// For more details on the configuration options, their defaults, and their associated environment variables, see
+    /// `OTel.Configuration`.
+    ///
+    /// If you need greater control over the bootstrap of the observability subsystems, use the APIs that return the
+    /// backends themselves, for example, `OTel.makeTracingBackend(configuration:)`.
```

Comment:
```suggestion
    /// backends themselves, for example, ``OTel/makeTracingBackend(configuration:)``.
```

You can use a docc link here.

Reply:
We've popped something on the roadmap issue for adding these links to the DocC docs. I tried to get them working both here and on the earlier PR to add docs to the configuration structs. I was having some issues getting them to resolve, which may (or may not) be because of the use of the same name for the enum and module.

We can discuss this at the next sync, too.